rpcs3/Emu/System.h

```diff
@@ -508,6 +508,7 @@ struct cfg_root : cfg::node
 		cfg::_bool downmix_to_2ch{this, "Downmix to Stereo", true};
 		cfg::_int<2, 128> frames{this, "Buffer Count", 32};
 		cfg::_int<1, 128> startt{this, "Start Threshold", 1};
+		cfg::_int<0, 100> volume{this, "Master volume", 100};
```

Comment:
The rest of the options are capitalized:
```suggestion
		cfg::_int<0, 100> volume{this, "Master Volume", 100};
```

Reply:
Some of them are capitalized and some of them are not, f.ex. "Dump to file" in start of the audio section and other examples include "Camera type", "Enter button assignment", "Log shader programs" and so on. But I guess uppercase is the more common variation.